content/technotes/2018-08-08-phylotar-release.md

```diff
@@ -0,0 +1,173 @@
+---
+slug: "phylotar"
+title: "phylotaR: Retrieve Orthologous Sequences from GenBank"
+package_version: 1.0.0
+authors:
+  - name: Dom Bennett
+    url: https://github.com/dombennett
+date: 2018-08-08
+categories: technote
+topicid: 1256
+tags:
+- r
+- ropensci
+- community
+- software
+- review
+- onboarding
+- package
+- evolution
+- bioinformatics
+- phylogenetics
+- genbank
+- dna
+- sequences
+- data-access
+- taxonomy
+---
+
+## What is phylotaR?
+
+In any phylogenetic analysis it is important to identify sequences that share the same orthology. This is often performed by searching an online sequence repository, such as GenBank, using a gene/region name and the taxa of interest, e.g. 'Aotus' and 'cytochrome oxidase'. Relying solely on sequence labels, however, can miss sequences that have either not been labelled, have unanticipated names or have been mislabelled.
+
+The phylotaR R package, like its earlier inspiration [PhyLoTa](http://phylota.net/), identifies sequences of shared orthology not by naming matching but instead through the use of an alignment search tool, [BLAST](https://en.wikipedia.org/wiki/BLAST). As a result, a user of phylotaR will able to download more relevant sequence data for their phylogenetic analysis than they otherwise would.
+
+The entire phylotaR pipeline is automated and all a user needs to supply is a taxonomic group of interest for which the user would like to identify ortholgous sequences. The pipeline is broken down into four stages: retrieve taxonomic information, download sequences, identify clusters, identify clusters among the clusters.
+
+![phylotaR pipeline stages](/img/blog-images/2018-08-08-phylotar/stages.png)
+
+For more information on how the pipeline works, please see the open-access scientific article: [phylotaR: An Automated Pipeline for Retrieving Orthologous DNA Sequences from GenBank in R](https://doi.org/10.3390/life8020020)
+
+
+## Installation
+
+phylotaR is available from CRAN.
+
+```r
+install.packages('phylotaR')
+```
+
+Alternatively, the latest development version can be downloaded from phylotaR's [GitHub page](https://github.com/ropensci/phylotaR).
+
+```r
+devtools::install_github(repo = 'ropensci/phylotaR', build_vignettes = TRUE)
+```
+
+### BLAST+
+
+In addition to the R package you will also need to have installed BLAST+ -- a local copy of the well-known [BLAST software](https://en.wikipedia.org/wiki/BLAST) on your machine. Unfortnately this can be a little complex as its installation depends on your operating system (Windows, Mac or Linux). Fortunately, NCBI provides detailed installation instructions for each flavour of operating system.
+
+**To install BLAST+ on your local machine follow the [NCBI installation instructions](https://www.ncbi.nlm.nih.gov/books/NBK279671/).**
```

Comment:
You might merge this sentence with the previous one to reduce text dedicated to installing BLAST since it requires effort but is not the objective of this post